relational_operators/TextScanOperator.cpp

```diff
@@ -234,11 +234,10 @@ void TextScanOperator::receiveFeedbackMessage(const WorkOrder::FeedbackMessage &
   }
 }
 
-
```

Comment:
Please reset this change, as we use two lines to separate methods from different classes. In this case, a operator and its work order. Thanks.